src/pages/studyView/virtualStudy/VirtualStudy.tsx

```diff
@@ -0,0 +1,248 @@
+                                        placeholder="Virtual study description (Optional)"
+                                        value={this.description}
+                                        onChange={event => this.description = event.currentTarget.value}
+                                    />
+                                </div>
+                                <span style={{ 'display': 'block', 'font-weight': 'bold' }}>This virtual study was derived from:</span>
+                                <div className={styles.studiesSummaryInfo}>
+                                    {
+                                        this.props.studyWithSamples.map(study => <StudySummaryRecord {...study} />)
+                                    }
+                                </div>
+                            </div>
+                        </Then>
+                        <Else>
+                            <div className={classnames(styles.virtualStudyResult)}>
+                                <a
+                                    target='_blank'
+                                    href={`${this.virtualStudyUrl}`}
+                                    style={{ 'margin-right': '10px' }}>
+                                    {this.virtualStudyUrl}
+                                </a>
+                                <div className="btn-group btn-group-xs">
+
+                                    <DefaultTooltip
+                                        overlay={
+                                            <If condition={this.copied}>
+                                                <Then>
+                                                    <span className="alert-success">Copied!</span>
+                                                </Then>
+                                                <Else>
+                                                    <span>Copy</span>
+                                                </Else>
+                                            </If>}
+                                        placement="top"
+                                        mouseLeaveDelay={0}
+                                        mouseEnterDelay={0.5}
+                                        onVisibleChange={this.onTooltipVisibleChange as any}
+                                    >
+                                        <span
+                                            className="btn btn-default"
+                                            ref={this.copyLinkRef}
+                                            onClick={this.onCopyClick}>
+                                            Copy
+                                        </span>
+                                    </DefaultTooltip>
+                                    <span
+                                        className="btn btn-default"
+                                        onClick={(event) => window.open(this.virtualStudyUrl, "_blank")}>
```

Comment:
Just leave a comment here: since the SV does not support VS yet, we need to link the View button to new study view later. And also need to fully test the content after saving VS base on VS.